ExampleGallery/Effects/VariableVideoEffect.cs

```diff
@@ -0,0 +1,84 @@
+﻿using Windows.Foundation.Collections;
```

Comment:
Please start the file with a standard copyright banner comment, the same as other Win2D source files